dataeng/resources/dbtsource-freshness.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -ex
+
+# Setup
+cd $WORKSPACE/warehouse-transforms
+# To install right version of dbt
+pip install -r tools/dbt_schema_builder/requirements.txt
+
+cd $DBT_PROJECT
+dbt clean
+
+cd $WORKSPACE/warehouse-transforms
+
+dbt source snapshot-freshness --project $DBT_PROJECT --profile $DBT_PROFILE --profiles-dir $WORKSPACE/analytics-secure/warehouse-transforms/
```

Comment:
I think you want this to be:

`dbt source snapshot-freshness --profile $DBT_PROFILE --target $DBT_TARGET --profiles-dir $WORKSPACE/analytics-secure/warehouse-transforms/`

So you would need to swap out the `DBT_PROJECT` param, which only Schema Builder cares about, for `DBT_TARGET`.

Reply:
Ok

Reply:
You might want to just run it locally to verify the cmd-line works. Also, it'd be nice to add at least a temporary [freshness check](https://docs.getdbt.com/docs/using-sources#section-snapshotting-source-data-freshness) to ensure that it succeeds/fails when expected - *then* run the cmd-line.

Reply:
Yeah i've tested it locally and it works. But since we haven't configured sources for dbt so we couldn't create any freshness check in dbt. 
I'll create a sample model and will test this.

Reply:
We actually *do* have sources - all the application DBs are accessible via `{{ source('LMS', 'AUTH_USER') }}` for example. So you could put freshness checks on any of those DBs/tables, I think.

Reply:
Here are 2 test runs for snapshot-freshness (for WARN & Error in freshness),
http://jenkins.analytics.edx.org/job/dbtsource-freshness/2/console
http://jenkins.analytics.edx.org/job/dbtsource-freshness/4/console